### PR TITLE
Fix issue writing json file to disk

### PR DESCRIPTION
**Pull request by BrettCawley**

### CODE REVIEW

**Comment by BrettCawley** [comment]
Fixes #2660

File: common/src/main/java/bisq/common/storage/JsonFileManager.java [hunk 1]
```diff
@@ -78,6 +78,7 @@ public void writeToDisc(String json, String fileName) {
 
                 printWriter = new PrintWriter(tempFile);
                 printWriter.println(json);
+                printWriter.close();
```
Comment (ManfredKarrer) [inline common/src/main/java/bisq/common/storage/JsonFileManager.java:81]: It is in the finally statement:
```
if (printWriter != null)
                    printWriter.close();
```

**Review by ManfredKarrer — COMMENTED** [review]
NACK

**Comment by ManfredKarrer** [comment]
I checked at another class and added some fix there which seems to have been needed on Windows. 
At the end its the same as you suggested but it comes with a comment why as the close is in the finally statment anyway. 
See https://github.com/bisq-network/bisq/pull/2669

**Pull request closed without merging**